config.go

```diff
@@ -50,6 +50,7 @@ type config struct {
 	deleteStateIndex bool // Remove state index on startup
 	initialOffset    int  // If creating initial state, start from this far back (seconds)
 	noSendAlert      bool // Don't send alerts to MozDef
+	overrides		[]Override // Keeps track of custom ip -> city,country overrides
```

Comment:
Sheesh, I even opened this file up in vim to try and match the tab vs spaces but apparently couldn't get it right.